tests/tests_pytorch/models/test_ddp_fork_amp.py

```diff
@@ -0,0 +1,41 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from torch.utils.data import DataLoader
+
+import tests_pytorch.helpers.utils as tutils
+from pytorch_lightning import Trainer
+from pytorch_lightning.demos.boring_classes import RandomDataset
+from tests_pytorch.helpers.runif import RunIf
+from tests_pytorch.helpers.test_models import AMPTestModel
+
+
+# needs to be standalone to avoid other processes initializing CUDA
+@RunIf(min_cuda_gpus_no_init=2, skip_windows=True, min_torch="1.12", standalone=True)
```

Comment:
Why is `min_torch` set?

Reply:
Good catch! Removed.
btw, this test won't run with torch == 1.12.0 because of the issue fixed [here](https://github.com/pytorch/pytorch/pull/80899/files#diff-dd8250f2740baa59fd4ba7431ad9d43f560d801ae2226ad492c2be9858d0b660) in 1.12.1 but I don't think that's an issue